Makefile

```diff
@@ -14,14 +14,16 @@ IMAGE_BUILDER?=docker
 IMAGE_BUILD_OPTS?=
 DOCKERFILE?=Dockerfile
 DOCKERFILE_CONFIG_DAEMON?=Dockerfile.sriov-network-config-daemon
+DOCKERFILE_WEBHOOK?=Dockerfile.webhook
 
 CRD_BASES=./config/crd/bases
 
 export APP_NAME?=sriov-network-operator
 TARGET=$(TARGET_DIR)/bin/$(APP_NAME)
 IMAGE_REPO?=ghcr.io/k8snetworkplumbingwg
 IMAGE_TAG?=$(IMAGE_REPO)/$(APP_NAME):latest
-CONFIG_DAEMON_IMAGE_TAG?=$(IMAGE_REPO)/sriov-network-config-daemon:latest
+CONFIG_DAEMON_IMAGE_TAG?=$(IMAGE_REPO)/$(APP_NAME)-config-daemon:latest
```

Comment:
Notice that this is changing from:
```
sriov-network-config-daemon
```
to:
```
sriov-network-operator-config-daemon
```

Maybe other things need to be updated as well. https://github.com/search?q=repo%3Ak8snetworkplumbingwg%2Fsriov-network-operator%20sriov-network-config-daemon&type=code

Reply:
image which is built part of our pipeline is named sriov-network-operator-config-daemon

see: https://github.com/k8snetworkplumbingwg/sriov-network-operator/pkgs/container/sriov-network-operator-config-daemon

the goal is to align makefile target to tag with same image name as we publish.

not aiming into renaming the component ATM :)